src/App.js

```diff
@@ -2,42 +2,93 @@ import './App.css';
 import GoblinForm from './GoblinForm';
 import GoblinList from './GoblinList';
 import Goblin from './Goblin';
+import { useState } from 'react';
+import { useEffect } from 'react/cjs/react.development';
```

Comment:
your netlify deploy is failing because of the `useEffect` import on line 6.  Should read `import { useEffect } from 'react' `  This is some kind of auto-import issue that a lot of people have been running into.